docs/contributing/coding-guidelines/ros-nodes/directory-structure.md

```diff
@@ -1,5 +1,42 @@
 # Directory structure
 
+## C++ package
+
+```txt
+<package-name>
+├─ config
+├─ include
+│   └─ <package-name>
+├─ launch
+├─ src
+├─ test
+├─ package.xml
+└─ CMakeLists.txt
+```
+
+### config directory
+
+Place configuration files such as node parameters.
+
+### include directory
+
+Place header files exposed to other packages under a directory with the package name. Do not place files directly under include directory.
+This is mostly library headers and such. Note that many headers do not need to be placed here.
+
+### launch directory
+
+Place launch files.
+
+### src directory
+
+Place source and private header files.
+
+### test directory
+
+Place source files for testing.
+
+## Python package
+
 !!! warning
 
     Under Construction
```

Comment:
Could you add content under this warning or remove the warning?

Reply:
Done https://github.com/autowarefoundation/autoware-documentation/pull/224/commits/763108b5b2b7440e9474bed2e33ecab472b4cc75.